pkg/cloud/azure/actuators/machine/reconciler.go

```diff
@@ -53,6 +53,18 @@ import (
 const (
 	// DefaultBootstrapTokenTTL default ttl for bootstrap token
 	DefaultBootstrapTokenTTL = 10 * time.Minute
+
+	// MachineRegionLabelName as annotation name for a machine region
+	MachineRegionLabelName = "machine.openshift.io/region"
+
+	// MachineAZLabelName as annotation name for a machine AZ
+	MachineAZLabelName = "machine.openshift.io/zone"
+
+	// MachineInstanceStateAnnotationName as annotation name for a machine instance state
+	MachineInstanceStateAnnotationName = "machine.openshift.io/instance-state"
```

Comment:
Once consolidated on aws this labels should be moved to machine controller. Also please add a comment to this annotation the likes of "The instance-state is a simple and cloud provider opinionated summary of what's the state of the machine,  this should not be relied upon for comprehensive observations

Reply:
How would machine controller set value of the annotation given it has no access to cloud provider specific information. Can you please describe more in detail what's the expected workflow of the annotation in machine controller generic part?

Reply:
just declare the labels there so they can be reused and set across the actuators instead of having to redeclared them every time